NotifySwiftQuickstart/ViewController.swift

```diff
@@ -10,11 +10,14 @@ import UIKit
 
 class ViewController: UIViewController {
   
-  var serverURL : String = "http://YOUR_SERVER_HERE/register"
+  var serverURL : String = "https://f107bbc9.ngrok.io"
```

Comment:
Please change this back to "YOUR_SERVER_HERE" url so users know to change to their own server source.